src/components/GenreMergeDialog.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <VDialog
+    ref="dialogMerge"
+    v-model="mergeModal"
+    v-if="isModerator"
+    width="600px"
+  >
+    <template v-slot:activator="{ on }">
+      <VBtn
+        v-on="on"
+        @click.stop.prevent
+        class="ma-2"
+        color="edit"
+        text
+        icon
+        small
+      >
+        <VIcon>mdi-merge</VIcon>
+      </VBtn>
+    </template>
+    <VCard>
+      <VCardTitle>
+        <span class="headline">{{
+          $t("music.genre.merge-into", { obj: genre.name })
+        }}</span>
+      </VCardTitle>
+      <VCardText>
+        <VContainer>
+          <VRow>
+            <VCol cols="12">
+              <VCombobox
+                :items="sortedGenres"
+                cache-items
+                chips
+                deletable-chips
```

Comment:
Tested this locally. I feel like these shouldn't be chips? If I understand correctly, we only want to be able to select one genre here, so a regular select seems more appropriate.

Reply:
you're right - i've updated